### DISCUSSION TITLE
Debugger does not debug through Calculation Group / Item Logic

### DISCUSSION THREAD

**Discussion by corycundy** [TabularEditor/TabularEditor3#1028] [Ideas]
It appears that the debugger does incorporate the logic contained within a Calculation Item, however, the debugger doesn't step through the DAX code within the Calculation Item.

I added a Calculation Group and Calculation Item with the following code to an Adventure Works DB and the debugger calculated the correct results, but code within the Calculation Item wasn't stepped through.

VAR M = SELECTEDMEASURE( ) RETURN IF( M <> BLANK( ), M * 2 )

I didn't see this posted as a limitation in the DAX debugger documentation.

**Comment by otykier** [comment]
This is a limitation that we have currently. We are evaluating internally how to visualize the presence of a calculation item in the filter context, which is not trivial. I've updated the docs to reflect this.
